src/libImaging/Storage.c

```diff
@@ -750,6 +753,17 @@ ImagingNewArrow(
             return im;
         }
     }
+    // Stored as [r,g,b,a,r,g,b,a....]
```

Comment:
```suggestion
    // Stored as [r,g,b,a,r,g,b,a...]
```
Nit: Three dots in an ellipsis

Reply:
The real issue is that the first period should be a comma.